From edbe0f36e2b953bfc0ceeef77439209838933abb Mon Sep 17 00:00:00 2001
From: Dylan Reid <dgreid@chromium.org>
Date: Tue, 3 Mar 2015 09:38:28 -0800
Subject: [PATCH 569/634] power: Remove HAS_WAKELOCK config and document
 WAKELOCK

Remove the HAS_WAKELOCK config as it doesn't seem to have been used in
the 3.10 or 3.14 kernels.

Add some Documentation to CONFIG_WAKELOCK so that it is selectable and
can be disabled is desired.

Signed-off-by: Dylan Reid <dgreid@chromium.org>
---
 kernel/power/Kconfig | 11 ++++++-----
 1 file changed, 6 insertions(+), 5 deletions(-)

diff --git a/kernel/power/Kconfig b/kernel/power/Kconfig
index 448b3f9..26bae40 100644
--- a/kernel/power/Kconfig
+++ b/kernel/power/Kconfig
@@ -18,13 +18,14 @@ config SUSPEND_FREEZER
 
 	  Turning OFF this setting is NOT recommended! If in doubt, say Y.
 
-config HAS_WAKELOCK
-	bool
-	default y
-
 config WAKELOCK
-	bool
+	bool "Android's method of preventing suspend"
 	default y
+	---help---
+	  This allows applications to prevent the CPU from suspending while
+	  they need it.
+
+	  Say Y if you are running an android userspace.
 
 config HIBERNATE_CALLBACKS
 	bool
-- 
1.9.1

